docker/docker_client.go

```diff
@@ -132,6 +178,15 @@ func newDockerClient(ctx *types.SystemContext, ref dockerReference, write bool)
 	}, nil
 }
 
+func hasFile(files []os.FileInfo, name string) bool {
```

Comment:
if `hasFile` is only used by `setupCertificates`, it would be a tiny bit more readable to have the two functions next to each other.  Do feel free to merge as is.